reference/conanfile/methods/build_requirements.rst

```diff
@@ -16,8 +16,8 @@ For simple cases the attribute syntax can be enough, like ``tool_requires = "cma
 
 The ``tool_requires`` and ``test_requires`` methods are just a specialized instance of ``requires`` with some predefined trait values. See the :ref:`requires() reference<reference_conanfile_methods_requirements>` for more information about traits.
 
-tool_requires
--------------
+tool_requires()
```

Comment:
This is wrong. The first level method is ``def build_requirements()``.
The ``tool_requires`` etc, are helpers like ``self.tool_requires()``, not recipe methods
This should be second level, not first level.